src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    Circle(Color color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void print() {
+        System.out.println("Figure: circle, area: " + getArea() + " sq. units, radius: "
+                + radius + " units, color: " + color);

```

Comment:
The `color` field is accessed directly here, which may lead to issues if the field is not visible. Ensure that `color` is accessible or use a getter method if one is available.